salt/docker/stop.sls

```diff
@@ -0,0 +1,4 @@
+# Stop and disable the docker daemon
+docker:
+  service.dead:
+    - enable: False
```

Comment:
Salt will "pick" the init system to use based on what it sees installed, so should use systemd unless there are some horrible bugs in salt ;)

Should we be stopping containerd before or after docker to avoid this? Ill update to do this explicitly.